github/testground.yml

```diff
@@ -215,6 +215,8 @@ teams:
         - KatarinaKeti
         - StefanMiletich
         - dektech
+        - StefanGajic
+        - bokimilinkovic
     privacy: closed
   maintainers:
     description: Testground maintainers
```

Comment:
@galargh I was looking for the github-mgmt option to `Require branches to be up to date before merging` on master.

I couldn't find it, so I enabled it [manually](https://github.com/testground/testground/settings/branch_protection_rules/14870337) and triggered a sync. It's not in this PR, which means it's not managed by our script, correct? (it couldn't be a latency issue or a mistake on my side)

Reply:
We currently ignore all the properties of branch protection rules. See https://github.com/testground/github-mgmt/blob/master/terraform/resources_override.tf#L36

If you want us to start managing `Require branches to be up to date before merging` through github management, simply remove https://github.com/testground/github-mgmt/blob/master/terraform/resources_override.tf#L45 out of the `ignored_changes` list (feel free to remove all the props from that list). Once that's done, you can either merge to master and then run sync on master or run sync on the branch with your change and then merge to master.

I tried to describe it in https://github.com/testground/github-mgmt/blob/master/docs/HOWTOS.md#start-managing-new-resource-attribute-through-github-management